### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by divinecharlotte**
## IN THIS PR I IMPLEMENTED THE FOLLOWING:

- [ ]  Create React application with react-app
- [ ]  Set up linter
- [ ]  Add routes to the project
- [ ]  Set up redux
- [ ]  Create an action reducer to fetch currency data from API
- [ ]  Add reducer to update list of store's currency
- [ ]  Create an action reducer to fetch currency details from API
- [ ]  Add reducer to update details of store's currency
- [ ]  Test project component, pages, reducers, pure functions and routes
- [ ]  Add styles to the project
- [ ]  Update readme file
- [ ]  Fix linter issues
### - while merging the testing file which was containing the required changes I accidentially merged it to the main branch
- [ ] here is the [loom video](https://www.loom.com/share/1b7d9fefeee54615829429ae6e6d6cc4) describing the project

### CODE REVIEW

File: src/redux/CoinList/CoinList.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+const FETCH_COINS = 'FETCH_COINS';
```
Comment (FrederickMih) [inline src/redux/CoinList/CoinList.js]: - [ ] Kindly scope your actions types names based on the feature they belong to. This helps when debugging more complex applications.

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+              handleDetails={handleDetails}
+            />
+          ))}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default Home;
```
Comment (FrederickMih) [inline src/pages/Home.js]: - [ ] Based on the requirements of this project the Home page has a list of categories that could be filtered using given parameters, e.g. category name. This should be a Filter stateless component to filter data retrieved from the API. Kindly implement this feature.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @divinecharlotte , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



# To Highlights

- The project has a Home and Detail page :+1: 


## Suggested changes

- [ ] `Video presentation:` Please could you kindly present to us the piece of code you found interesting?. In the requirements of this project, you were asked to write unit tests for several components. If those tests were done in a kind of interesting way, then put more lights on it.
- [ ] I would once more suggest that you should fix the deployment issues.

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by elyor-doniyorov — APPROVED** [review]
Hi @divinecharlotte, 

In my opinion, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 
Approved-d house


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**